test-assets/role-manifests/app/tor-validation-ok.yml

```diff
@@ -2,15 +2,8 @@
 ---
 instance_groups:
 - name: myrole
-  environment_scripts:
-  - environ.sh
-  - /environ/script/with/absolute/path.sh
   scripts:
   - myrole.sh
-  - /script/with/absolute/path.sh
-  post_config_scripts:
-  - post_config_script.sh
-  - /var/vcap/jobs/myrole/pre-start
```

Comment:
Probably leave the absolute paths in? Unless you have the equivalent checks elsewhere…

Reply:
Yes, these are tested for explicitly elsewhere.
They were copy pasted I assume in here, and they don't add to the tests where this is used.